src/store/modules/events.js

```diff
@@ -15,7 +15,6 @@ export default {
       // slice truncates any out-of-bounds indices, so that's not a concern here
       return state.events.slice(firstEvent, lastEvent);
     },
-    numOfEvents: state => state.events.length,
     getEventById: state => id => state.events.find(event => event.id === id),
     isRegistered: state => id => !!state.myEvents.find(event => event.id === id),
```

Comment:
It should be safe to remove this right? If not we should replace those uses as well

Reply:
So can you just remove it in this PR?